src/_common.scss

```diff
@@ -14,12 +14,21 @@
     src: local('Lato Bold'), local('Lato-Bold'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff2') format('woff2'), url('https://cdn01.boxcdn.net/fonts/1.0.2/lato/Lato-Bold.woff') format('woff');
 }
 
+.flyout-overlay {
+    @import '~box-react-ui/lib/styles/common/overlay';
+    @import '~box-react-ui/lib/styles/common/links';
+    @import '~box-react-ui/lib/styles/common/buttons';
+}
+
 .ba {
     @include reset;
 
+    /* stylelint-disable no-duplicate-at-import-rules */
```

Comment:
is there a way to avoid the duplication?

Reply:
The flyout is a component which attaches to the body rather than our preview so at least scoping it to only be imported inside the flyout-overlay is better than generally importing :-/